jupyter/datascience/ubi9-python-3.11/Dockerfile

```diff
@@ -11,6 +11,19 @@ LABEL name="odh-notebook-jupyter-datascience-ubi9-python-3.11" \
     io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/jupyter/datascience/ubi9-python-3.11" \
     io.openshift.build.image="quay.io/opendatahub/workbench-images:jupyter-datascience-ubi9-python-3.11"
 
+WORKDIR /opt/app-root/opt
```

Comment:
I believe this is the largest question right now; where to locate the _other_ venv

Reply:
`/opt/app-root/opt` is poor choice